packages/patternfly-4/react-core/src/components/TextInput/TextInput.js

```diff
@@ -50,6 +52,7 @@ const defaultProps = {
   isDisabled: false,
   isReadOnly: false,
   type: 'text',
+  placeholder: '',
```

Comment:
Shouldn't need to add this. All input props are spread to the I put element. Placeholder will just work.

Reply:
Checked, I don't know why not worked for me. Thanks

Reply:
I think we should take a look and figure out why the spreading of props is not working.

Reply:
When using the LoginForm, the props are only spread to the Form, not the inner TextInput. So looks like we need to have these additional placeholder props on the LoginForm, or add more general userNameProps, passwordProps that are passed along

Reply:
It's working @tlabaj , I said didn't know why not worked but it's working with this PR see the build. This is outdate